jaeger-core/src/main/java/io/jaegertracing/internal/propagation/TextMapCodec.java

```diff
@@ -83,7 +91,7 @@ public JaegerSpanContext extract(TextMap carrier) {
     }
     if (context == null) {
       if (debugId != null) {
-        return JaegerSpanContext.withDebugId(debugId);
```

Comment:
so you're not even using that withDebugId method! Kill it.

Reply:
Definitely.